docs/changes.rst

```diff
@@ -1,5 +1,10 @@
 .. _changes:
 
+4.5.3 (2018-12-16)
```

Comment:
Please leave me the privilege to decide when the next release will be published.

XXX for a new version and date would be fine as placeholders.

Reply:
Oh, sorry, I misunderstand you and did more than needed.